src/app/blog/utils.ts

```diff
@@ -15,12 +19,16 @@ export interface Post {
   publishedDate: string;
 }
 
-export async function getPostSlugs(): Promise<string[]> {
+export async function getStaticPostSlugs(): Promise<string[]> {
   const fileNames = fs.readdirSync(POSTS_FILE_PATH);
   return fileNames.map((fileName) => fileName.replace(".md", ""));
 }
 
-export async function getPostBySlug(slug: string): Promise<Post> {
+/**
+ * Get a static post using slug.
+ * @param slug URL slug string. Should be a non-numerical string.
+ */
+export async function getStaticPostBySlug(slug: string): Promise<Post> {
   const readFile = fs.readFileSync(`${POSTS_FILE_PATH}/${slug}.md`, "utf-8");
   const {
     data: {
```

Comment:
Should we remove all of this static stuff now? (And update wording of "remote" stuff to just blog/post